src/elements/form/DynamicForm.js

```diff
@@ -81,7 +81,7 @@ export class NovoDynamicFormElement {
         return this.form ? this.form.valid : false;
     }
 
-    get updatedValues() {
+    updatedValues() {
```

Comment:
Was called literally every dirty check.. now this isn't referenced in the templates in the app and just invoked